codes/ruby/chapter_searching/two_sum.rb

```diff
@@ -0,0 +1,45 @@
+=begin
+File: two_sum.rb
+Created Time: 2024-04-09
+Author: Blue Bean (lonnnnnnner@gmail.com)
+=end
+
+### 方法一：暴力枚举 ###
+def two_sum_brute_force(nums, target)
+  # 两层循环，时间复杂度为 O(n^2)
+  for i in 0...nums.length
```

Comment:
In the python code, the for loop is

```py
for i in range(len(nums) - 1):
```

It traverse the `nums` in range [0, nums.length - 1).

In your code, the loop traverses in range [0, nums.length). So please change to this to have a correct range:

```suggestion
  for i in 0..nums.length
```

Reply:
0..nums.lenth seems to traverse the nums in [0, nums.length]. I will repair it, thank you!

Reply:
Oops! I got a mistake in the suggestion code. It should be 

```suggestion
  for i in 0...(nums.length - 1)
```

Sorry for the misinformation :smiley:

Reply:
I have made all the changes in commit
369f7e3